bokeh/models/actions.py

```diff
@@ -0,0 +1,17 @@
+""" Client-side interactivity. """
+
+from __future__ import absolute_import
+
+from ..plot_object import PlotObject
+from ..properties import String
+
+class Action(PlotObject):
+    """ Base class for interactive actions. """
+
+class OpenURL(Action):
+    """ Open a URL in a new tab or window (browser dependent). """
+
+    url = String("http://", help="""
+    The URL to direct the web browser to. This can be a template string,
+    which will be formatted with data from the data source.
+    """)
```

Comment:
I wonder what other dead simple actions would could add that would have the most impact. We will be having almar work on a function interface right after 0.8 (that could potentially just be a "JSAction" but there are surely lots of other useful "canned" actions. @pzwang thoughts?